src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx

```diff
@@ -33,27 +22,14 @@ const WinterStorageNotice = () => {
       <Row>
         <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
           <h3>{t('page.winter_storage.notice.title')}</h3>
+          <p>{t('page.winter_storage.notice.paragraph1')}</p>
+          <p>{t('page.winter_storage.notice.paragraph2')}</p>
           <p>
             <Trans
-              i18nKey={'page.winter_storage.notice.paragraph1'}
-              components={[
-                <a
-                  key={'wsNoticeParagraph1Link'}
-                  href={getBerthReservationsLink(language)}
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  content
-                </a>,
-              ]}
-            />
-          </p>
-          <p>
-            <Trans
-              i18nKey={'page.winter_storage.notice.paragraph2'}
+              i18nKey={'page.winter_storage.notice.paragraph3'}
```

Comment:
It is superhard to follow what the content actually is. I think there could be some hints about the context at least. The Trans tag has for example a [default attribute](https://www.hel.fi/en/culture-and-leisure/outdoor-activities-parks-and-nature-destinations/boating/winter-storage-of-boats).